packages/forms/resources/views/components/textarea.blade.php

```diff
@@ -44,8 +44,10 @@
                 x-data="textareaFormComponent({ initialHeight: @js($initialHeight) })"
                 x-ignore
                 x-intersect.once="render()"
+                x-on:form-validation-error.window="$nextTick(() => render())"
```

Comment:
Is this line really needed, @danharrin @emamalias?

Reply:
I'm not sure if we can remove `x-intersect` yet, but both attributes are not needed :) seems like the `wire:ignore.style.height` does the work. I'll make the change. thanks

Reply:
Tested with this:

```php
Forms\Components\Textarea::make('Test Textarea')
        ->prohibited()
        //->live(debounce: 500)
        ->afterStateUpdated(fn (Forms\Components\Textarea $component) =>
            $component->isLive() && $this->validateOnly($component->getStatePath())
        )
        ->autosize()
``` 
Works with/without live validation


https://github.com/filamentphp/filament/assets/13638285/45963199-3050-40e0-acc8-be9f4bbb8dc6

#